x-pack/metricbeat/module/aws/sqs/_meta/fields.yml

```diff
@@ -0,0 +1,10 @@
+- name: sqs
+  type: group
+  description: >
+    `sqs` contains the metrics that were scraped from AWS CloudWatch which contains monitoring metrics sent by AWS SQS.
+  release: beta
+  fields:
+    - name: oldest_message_age
+      type: scaled_float
+      description: >
+        The approximate age of the oldest non-deleted message in the queue.
```

Comment:
what is the unit here? If it's sec, I wonder if we should make this just a long. I worry with scaled_float if the value becomes very large, it will be inaccurate.

Reply:
One trick we could do for such values is, using `long` here but still send up a float. What happens then (I think but we need to test) is that in `_source` the original value will be but for indexing purpose ES will convert it to a long. This means we can have simple queries but still have the original value around in case we need it.

Reply:
Yeah, the unit must be added to the description and the field itself: `oldest_message_age.sec` for example.

Reply:
Unit is sec here, based on the very few data points I see on cloudwatch, `long` should be the type here. Thanks!